backend/counterfactual/simulation.py

```diff
@@ -256,7 +256,9 @@ def simulate_dataframes(
 
         # Add cumulative cases that occurred before the simulation start date
         df_counterfactuals.append(
-            add_cumulative_sum(df_counterfactual_country, initial_cum_cases)
+            add_cumulative_sum(
+                df_counterfactual_country, initial_cum_cases, simulation_start_date
+            )
```

Comment:
I think that the first date in `df_counterfactual_country` will always be `simulation_start_date` so I'm not sure that this change is needed. Maybe I'm missing something though?

Reply:
Not really, df_counterfactual_country has the same date range that the real data dataframes (starting from January), but the modelling only happens in a subset of those ranges. Having different date ranges breaks the front end histogram that expects the data to have equal shapes. This shouldn't be an issue in most cases, but in some special countries like Norway not adding this step makes the histogram look like in issue #70

Reply:
So `df_counterfactual_country` comes from calling `simulate_single_country` which you pass `(simulation_start_date, simulation_end_date)` to. On line 320, you can see that the `simulation_dates` date range being used in this function runs from `simulation_start_date` to `simulation_end_date`. This is a change that I made when I was pulling these functions out into a separate function, so if it's wrong then feel free to revert to the previous behaviour.

Reply:
Yes, but in line 381 the resulting simulation dataframe gets outer-merged with the real case data which extends the date range, and all non common dates get filled with 0 (which would create the problem in Norway).

I don't think is wrong, in fact it is the same behaviour we have in the previous version.